online-dashboard-api/database/migrations/2024_12_25_093740_create_projects_table.php

```diff
@@ -13,9 +13,18 @@ public function up(): void
     {
         Schema::create('projects', function (Blueprint $table) {
             $table->id();
```

Comment:
Just for future purpose.
Never update the old migration! As it wont be reflecting unless and until we rollback everything!
If you need to add a new columns - just add a new migration!